src/data/ua_overrides.js

```diff
@@ -792,6 +792,35 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1751232 - Add override for sites returning desktop layout for Android 12
+     * Webcompat issue #92978 - https://github.com/webcompat/web-bugs/issues/92978
+     *
+     * A number of news sites returns desktop layout for Android 12 only. Changing it
+     * to Android 12.0 fixes the issue
+     */
+    id: "bug1751232",
+    platform: "android",
+    domain: "Sites with desktop layout for Android 12",
+    bug: "1751232",
+    config: {
+      matches: [
+        "*://*.dw.com/*",
+        "*://*.abc10.com/*",
+        "*://*.wnep.com/*",
+        "*://*.dn.se/*",
+        "*://*.dailymail.co.uk/*",
+      ],
+      uaTransformer: originalUA => {
+        if (!originalUA.includes("Android 12;")) {
+          return originalUA;
+        }
+
+        return originalUA.replace("Android 12;", "Android 12.0;");
```

Comment:
This almost feels worth writing a blog post about it, because it's a very ... surprising and interesting issue. Do you have a blog and want to write about it? :D

Reply:
I don't, maybe it's time to start one 😄 If you feel like writing about it, go ahead! 
I've noticed these sites have one common element as they all are news sites, so that's interesting :)